README.md

```diff
@@ -87,19 +87,19 @@ nlsolve(f!, initial_x)
 ```
 
 Alternatively, you can construct an object of type
-`DifferentiableMultivariateFunction` and pass it to `nlsolve`, as in:
+`DifferentiableVector` and pass it to `nlsolve`, as in:
 
 ```jl
-df = DifferentiableMultivariateFunction(f!)
+df = DifferentiableVector(f!)
 nlsolve(df, initial_x)
 ```
 
 If your function `f!` operates not on `AbstractVector` but on arbitrary
 `AbstractArray` you have to specify `initial_x` in the constructor of the
-`DifferentiableMultivariateFunction`:
+`DifferentiableVector`:
```

Comment:
this makes it a confusing name. So it can handle non-vector arrays?

Reply:
I get your point, requesting bikeshed backup immediately ;)